lms/static/sass/shared/_header.scss

```diff
@@ -377,6 +388,19 @@ header.global-new {
 
   .nav-global {
     @include float(left);
+    .tab-nav-item{
+      .tab-nav-link{
+        height: 43px;
+        font-size: em(16);
+        color: $gray;
+        &:hover{
```

Comment:
@AlasdairSwan I have added and tried to have focus here. But the UI looked super ugly. The blue outline when focus is on would mix with the blue underline. With two different blue colors, it was not good looking. Do we really want to have focus to behave like that?